CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Fixes an issue in **format** command where the `-i` option included files in `.venv` directories.
 * Updated the comment added to contribution PRs to old packs so it contains a link to the documentation of the **GitHub Codespaces** in xsoar.pan.dev.
 * Updated GitPython version to 3.1.32.
+* Fixed an issue where **update-release-notes** was reporting incorrectly that XSIAM event collectors were available from Cortex XSOAR instead of Cortex XSIAM.
```

Comment:
```suggestion
* Fixed an issue where **update-release-notes** generated "available from Cortex XSOAR" instead of "from XSIAM" when run on XSIAM event collectors.
```